CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [3.3.1] - 2022-05-18
+
+- Added support for SAML in sdk
+- Fixed issue where UMD bundle namespace changed
```

Comment:
Just noticed this. Do we want to be consistent with separating the lists "Added" and "Fixed" or are we just going for a single list?